src/components/gearvr-controls.js

```diff
@@ -0,0 +1,201 @@
+var registerComponent = require('../core/component').registerComponent;
```

Comment:
I think we should back the GearVR pieces until the GearVR controllers are released, then we can focus on Daydream for this PR.